### DISCUSSION TITLE
Formula Request: Erlang Language Platform

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#5569] [Writing Formulae/Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.17
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3426911432b93dc46d20d910c2f4fd33cdac62a2
Last commit: 4 days ago
Core tap JSON: 23 Aug 07:17 UTC
Core cask tap JSON: 23 Aug 07:17 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: octa-core 64-bit arm_ibiza
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

As the title says. The [Erlang Language Platform](https://github.com/WhatsApp/erlang-language-platform) looks _really_ nice, but sadly the lack of an `opam`-like for Erlang makes it a pain to retrieve (it's actually not even on their project-local package manager). I'll have a crack at packaging it myself at some point, but formula options seem to be _very_ varied, such that working from a template will yield bad results; I'm nervous to push something broken to Core.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by SMillerDev** [comment]
>  I'm nervous to push something broken to Core.

That's what we have human review and extensive CI builds for.

**Comment by robertoaloi** [comment]
Attempting in https://github.com/Homebrew/homebrew-core/pull/247432
